roles/jd-server/src/lib/mod.rs

```diff
@@ -35,6 +44,124 @@ pub fn get_coinbase_output(config: &Configuration) -> Result<Vec<TxOut>, Error>
     }
 }
 
+pub fn start_jd_server(config: Configuration) -> Receiver<status::Status> {
+    let url = config.core_rpc_url.clone() + ":" + &config.core_rpc_port.clone().to_string();
+    let username = config.core_rpc_user.clone();
+    let password = config.core_rpc_pass.clone();
+    // TODO should we manage what to do when the limit is reaced?
+    let (new_block_sender, new_block_receiver): (Sender<String>, Receiver<String>) = bounded(10);
+    let mempool = Arc::new(Mutex::new(mempool::JDsMempool::new(
```

Comment:
Maybe a `good first issue` to parse/validate and pass around an actual URL type instead of `String`.